modules/dmrpp_module/DmrppCommon.cc

```diff
@@ -91,11 +87,9 @@ void join_threads(pthread_t threads[], unsigned int num_threads)
             string *error = NULL;
             if ((status = pthread_join(threads[i], (void **) &error)) < 0) {
                 BESDEBUG(dmrpp_3, "Could not join thread " << i << ", " << strerror(status)<< endl);
-                // ERROR_LOG("Failed to join thread " << i << " during clean up from an exception: " << strerror(status) << endl);
```

Comment:
These calls to ERROR_LOG are crucial to understanding what is happening with this code when it's deployed in the filed, specifically NGAP. I commented them out because things seemed ok, but I think we should leave them in just in case we have issues down the road. They're the right things in the right place and it's not always easy to to sort that out from scratch every time